### PR TITLE
feat: init app-layout components

### PR DESCRIPTION
**Pull request by altaywtf**
# Header
image

image

# Footer
image

# AppLayout
image

### CODE REVIEW

**Comment by altaywtf** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9** <a href="https://app.graphite.dev/github/pr/okanisildar/beecast/9?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/okanisildar/beecast/9?utm_source=stack-comment).

File: .storybook/main.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ const config: StorybookConfig = {
     name: '@storybook/nextjs',
     options: {},
   },
+  staticDirs: ['../public'],
```
Comment (altaywtf) [inline .storybook/main.ts:17]: required for storybook to load stuff

File: components/layout/app-footer.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { GitHubLogoIcon, TwitterLogoIcon } from '@radix-ui/react-icons';
+import { Flex, Link, Text } from '@radix-ui/themes';
+import React from 'react';
+
+import { AppContent } from './app-content';
+
+const FOOTER_COMPANY_LINKS = [
+  {
+    href: 'https://experiment.st',
```
Comment (altaywtf) [inline components/layout/app-footer.tsx:9]: https://linear.app/experiment-station/issue/SUPA-27/add-privacy-policy-and-terms-of-service

File: components/layout/app-footer.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+] as const;
+
+export function AppFooter() {
+  return (
+    <AppContent>
+      <Flex justify="between" py="4">
+        <Flex align="start" direction="column">
+          <Text size="2" weight="bold">
+            Built with ☕️ by your folks at the{' '}
+            <Link color="gray" href="https://experiment.st" target="_blank">
```
Comment (altaywtf) [inline components/layout/app-footer.tsx:39]: https://linear.app/experiment-station/issue/SUPA-28/basic-homepage-for-experimentst

File: components/layout/app-footer.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  },
+  {
+    href: 'https://experiment.st',
+    label: 'Terms of service',
+  },
+] as const;
+
+const FOOTER_SOCIAL_LINKS = [
+  {
+    href: 'https://github.com/okanisildar/beecast',
```
Comment (okanisildar) [inline components/layout/app-footer.tsx:21]: 🙃
```suggestion
    href: 'https://github.com/experiment-station/beecast',
```

**Review by okanisildar — APPROVED** [review]

**Pull request merged**